tools_init/readme.md

```diff
@@ -19,21 +26,30 @@ Requirements
   - Sonar URL (`SONAR_URL` variable) : put here your custom Sonar URL ("http://localhost:9000" by default)
   - name of tag to add (`TAG_ECOCONCEPTION` variable) : the name of the new tag to add to a list of rules
   - rules keys list (string format separated with one comma) (`RULES_KEYS` variable) : specify here the list of all keys of rules that you want to add th new tag
-  
+  - name of profile to add (`PROFILE_ECOCONCEPTION` variable) : the name of the new profile to add for each language
+  - language keys list (string format separated with one comma) (`PROFILES_LANGUAGE_KEYS` variable) : specify here the list of all keys language that you want to add the new ecocode quality profile
+
```

Comment:
I think you would tell that a SonarQube token with "User" rights is mandatory (because with "general" rights will produce authorization errors)

Reply:
I  was thinking permission is a global problematic of this package scripts (tag + profile).
 Needed  to put   ' admininister Quality Profiles' to update tag and profile at the top of the readme.

Reply:
ok with you. can you add it in documentation ? thank you